regolith/builders/grantreportbuilder.py

```diff
@@ -90,13 +91,32 @@ def latex(self):
                                      types=["all"], since=rp_start_date, before=rp_end_date, statuses=["accepted"]))
         # thesis defendings
         # how do i access people.yml in rg-db-public vs the people.yml file in rg-db-group?
+        defended_theses = []
         for id in grant_people:
             person = self.gtx['people'][id]
-            if person['']
-        # How have results been disseminated
+            for education in person['education']:
+                if 'phd' in education['degree'].lower() and 'columbia' in education['institution'].lower() and \
+                        rp_start_date.year <= education['end_year'] <= rp_end_date.year:
```

Comment:
since you will be doing something like this a lot I guess, you may want to use a function.  Can you use is_current which has already been written?  I would probably try and use get_dates to get dates and then is_current.

It may not work, but using functions that are well tested and can handle edge cases well will save time later.